katalog/kubeadm-sm/rules.yml

```diff
@@ -8,53 +8,6 @@ metadata:
   namespace: monitoring
 spec:
   groups:
-  - name: kube-scheduler.rules
```

Comment:
Why were those rules removed from here?

Rules and alerts related to `controller-manager`, `scheduler` (and maybe `kube-proxy` too) makes sense only in unmanaged clusters since otherwise we don't have access to the needed metrics.

Can you check if in `prometheus-operated/rules.yml` there are rules related to control-plane components different from the apiserver?

Thank you.

Reply:
> Can you check if in prometheus-operated/rules.yml there are rules related to control-plane components different from the apiserver?

Yes there are. Seems like were "moved" to the prometheus-operated package.

Reply:
Please, take a look to: https://github.com/sighupio/fury-kubernetes-monitoring/pull/23/commits/1dab2940e39961961fb682b97d801df71ca70608 Thanks!